opba-api-security/src/main/java/de/adorsys/opba/api/security/service/RequestSigningService.java

```diff
@@ -0,0 +1,7 @@
+package de.adorsys.opba.api.security.service;
+
+import de.adorsys.opba.api.security.domain.SignData;
+
+public interface RequestSigningService {
+    String sign(SignData signData);
```

Comment:
Probably the name of the method is 
`
String signature(SignData signData);
`

Reply:
done